changes/fix_GLT-4245

```diff
@@ -0,0 +1 @@
+TAT trend report grouping logic
```

Comment:
I'm not sure what this means? What feature was broken and is now fixed? Colour by gate?